addons/bus/websocket.py

```diff
@@ -275,6 +279,7 @@ def get_messages(self):
                         else CloseCode.KEEP_ALIVE_TIMEOUT
                     )
                     continue
+                self._trim_sent_notifs()
```

Comment:
Wouldn't it make more sense to do it at the start of `_dispatch_bus_notifications`?
As it's not useful anywhere else. It clears the list from memory faster if there is no dispatch, at the cost of running the code way more often.

Reply:
Yeah you are right.